.env.sample

```diff
@@ -0,0 +1,17 @@
+# The `.env` file is used by our front-end development tooling to personalize
+# the behavior of development tasks. If you wish to alter the way our toolchain
+# behaves during development, copy the contents of this file into a `.env` file.
```

Comment:
Does it matter where it is placed?

Reply:
Marking this resolved while we iron out the details in [this other comment thread](https://github.com/ChromaticHQ/calliope/pull/5/files#r816765029).